tests/page-objects/users/users.wdio.page.js

```diff
@@ -0,0 +1,36 @@
+const utils = require('../../utils');
```

Comment:
As above, please merge the contents of this file with https://github.com/medic/cht-core/blob/master/tests/page-objects/admin/user.wdio.page.js

Reply:
removed this file and used existing one